### PR TITLE
Clarify Ubuntu Pro subscription requirement for Anbox Cloud

### PR DESCRIPTION
**Pull request by keirthana**
Modify warning messages to reflect the full Ubuntu Pro requirement for Anbox Cloud based on the current offerings.

Signed-off-by: Keirthana T S <keirthana.ts@canonical.com>

### CODE REVIEW

File: ref/glossary.md [hunk 1]
```diff
@@ -435,7 +435,7 @@ A server that finds the most optimal network path between a client and the conta
 <a name="ubuntu-pro"></a>
 ### Ubuntu Pro
 
-Canonical’s service package for Ubuntu that provides enterprise security and support for open-source applications, with managed service offerings available. Note the difference between Ubuntu Pro (Infra-only) and Ubuntu Pro (Apps-only); Anbox Cloud requires a full Ubuntu Pro or a Ubuntu Pro (Apps-only) subscription.
+Canonical’s service package for Ubuntu that provides enterprise security and support for open-source applications, with managed service offerings available. Note the difference between Ubuntu Pro (Infra-only) and full Ubuntu Pro subscriptions. Anbox Cloud requires a full Ubuntu Pro subscription.
 
 See [Ubuntu Pro](https://ubuntu.com/support).
```
Comment (ru-fu) [inline ref/glossary.md:441]: There's one more occurrence of "Ubuntu Pro (App-only)" just underneath in the "Ubuntu One" section.

**Review by ru-fu — COMMENTED** [review]
One small comment, otherwise this looks great!

File: ref/glossary.md (hunk 1 above)
Comment (keirthana) [reply ref/glossary.md:441]: Thank you for catching that, I have addressed this in 52e862e81bc59cb66b72ef35126e7b6d0798ba1b.

File: howto/install-appliance/azure.md [hunk 2]
```diff
@@ -11,8 +11,8 @@ Check the hardware requirements for the Anbox Cloud Appliance [here](https://dis
 In addition, make sure you have the following prerequisites:
 
 * An Ubuntu SSO account. If you don't have one yet, create it [here](https://login.ubuntu.com).
-* Your Ubuntu Pro token for either a full Ubuntu Pro subscription or a Ubuntu Pro (Apps-only) subscription. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a valid Ubuntu Pro token, sign in on https://ubuntu.com/pro to retrieve it.
-  [note type="caution" status="Warning"]The *Ubuntu Pro (Infra-only)* token that every user gets for free does **NOT** work and will result in a failed deployment. You must purchase a full *Ubuntu Pro* or a *Ubuntu Pro (Apps-only)* subscription by [contacting Canonical](https://anbox-cloud.io/contact-us).[/note]
+* Your Ubuntu Pro token for a full Ubuntu Pro subscription. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a valid Ubuntu Pro token, log in to https://ubuntu.com/pro to retrieve it.
```
Comment (morphis) [inline howto/install-appliance/azure.md:14]: I think we can drop `full` in here and only say `for a Ubuntu Pro subscription`

**Review by morphis — CHANGES_REQUESTED** [review]

File: howto/install-appliance/azure.md (hunk 2 above)
Comment (keirthana) [reply howto/install-appliance/azure.md]: > I think we can drop `full` in here and only say `for a Ubuntu Pro subscription`

@morphis I have fixed all occurrences of 'full Ubuntu Pro subscription'. Please have a look.

File: howto/install-appliance/azure.md [hunk 3]
```diff
@@ -11,8 +11,8 @@ Check the hardware requirements for the Anbox Cloud Appliance [here](https://dis
...
 
 * An Ubuntu SSO account. If you don't have one yet, create it [here](https://login.ubuntu.com).
-* Your Ubuntu Pro token for either a full Ubuntu Pro subscription or a Ubuntu Pro (Apps-only) subscription. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a valid Ubuntu Pro token, sign in on https://ubuntu.com/pro to retrieve it.
-  [note type="caution" status="Warning"]The *Ubuntu Pro (Infra-only)* token that every user gets for free does **NOT** work and will result in a failed deployment. You must purchase a full *Ubuntu Pro* or a *Ubuntu Pro (Apps-only)* subscription by [contacting Canonical](https://anbox-cloud.io/contact-us).[/note]
+* Your Ubuntu Pro token for an Ubuntu Pro subscription. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a valid Ubuntu Pro token, log in to https://ubuntu.com/pro to retrieve it.
+  [note type="caution" status="Warning"]The *Ubuntu Pro (Infra-only)* token does **NOT** work and will result in a failed deployment. You need a full *Ubuntu Pro* subscription.[/note]
```
Comment (morphis) [inline howto/install-appliance/azure.md:15]: Shouldn't we drop the `full`word in here?

File: howto/install/deploy-bare-metal.md [hunk 4]
```diff
@@ -7,8 +7,8 @@ Alternatively, you can follow the instructions in this document to use the [manu
...
-* Your Ubuntu Pro token for either a full Ubuntu Pro subscription or a Ubuntu Pro (Apps-only) subscription. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a valid Ubuntu Pro token, sign in on https://ubuntu.com/pro to retrieve it.
-  [note type="caution" status="Warning"]The *Ubuntu Pro (Infra-only)* token that every user gets for free does **NOT** work and will result in a failed deployment. You must purchase a full *Ubuntu Pro* or a *Ubuntu Pro (Apps-only)* subscription by [contacting Canonical](https://anbox-cloud.io/contact-us).[/note]
+* Your Ubuntu Pro token for an Ubuntu Pro subscription. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a valid Ubuntu Pro token, log in to https://ubuntu.com/pro to retrieve it.
+  [note type="caution" status="Warning"]The *Ubuntu Pro (Infra-only)* token does **NOT** work and will result in a failed deployment. You need a full *Ubuntu Pro* subscription.[/note]
```
Comment (morphis) [inline howto/install/deploy-bare-metal.md:11]: See above

File: howto/install/deploy-juju.md [hunk 5]
```diff
@@ -61,7 +61,7 @@ Every deployment of Anbox Cloud must be attached to the Ubuntu Pro service Canon
 
 You can retrieve your Ubuntu Pro token at https://ubuntu.com/pro after logging in. You should record the token as you will need it for every deployment of Anbox Cloud.
 
-[note type="caution" status="Warning"]The *Ubuntu Pro (Infra-only)* token that every user gets for free does **NOT** work and will result in a failed deployment. You must purchase a full *Ubuntu Pro* or a *Ubuntu Pro (Apps-only)* subscription by [contacting Canonical](https://anbox-cloud.io/contact-us).[/note]
+[note type="caution" status="Warning"]The *Ubuntu Pro (Infra-only)* token will result in a failed deployment. You need a full *Ubuntu Pro* subscription.[/note]
```
Comment (morphis) [inline howto/install/deploy-juju.md:64]: See above

**Review by morphis — CHANGES_REQUESTED** [review]

**Review by morphis — APPROVED** [review]
LGTM

File: howto/install-appliance/azure.md (hunk 3 above)
Comment (keirthana) [reply howto/install-appliance/azure.md]: Done

File: howto/install/deploy-juju.md (hunk 5 above)
Comment (keirthana) [reply howto/install/deploy-juju.md]: Done

File: howto/install/deploy-bare-metal.md (hunk 4 above)
Comment (keirthana) [reply howto/install/deploy-bare-metal.md]: Done

**Pull request merged**